01-JS-DOM-APIs/Ejercicios 11,12/css/style.css

```diff
@@ -0,0 +1,8 @@
+* {
+    background-color: white;
+    margin: 15px;
+}
+
+.matrixTable, .matrixTable--row2, .matrixTable--row2 {
```

Comment:
Why `.matrixTable--row2` two times?
Also use `-` to separate words. So `matrix-table` `matrix-table--row`